components/style/media_queries.rs

```diff
@@ -269,17 +275,16 @@ pub fn parse_media_query_list(input: &mut Parser) -> MediaList {
                 Err(()) => break,
             }
         }
-        media_queries
-    };
-    MediaList { media_queries: queries }
+        MediaList { media_queries: media_queries }
+    }
 }
 
 impl MediaList {
     pub fn evaluate(&self, device: &Device) -> bool {
         let viewport_size = device.au_viewport_size();
 
-        // Check if any queries match (OR condition)
-        self.media_queries.iter().any(|mq| {
+        // Check if it is an empty media query list or any queries match (OR condition)
```

Comment:
I don't think Servo currently exposes MediaList in DOM... CSSOM isn't the right spec to link here.

If we really want a link to spec, it should probably be [Media Queries 4](https://drafts.csswg.org/mediaqueries-4/#mq-list).